antora.yml

```
@@ -25,7 +25,7 @@ asciidoc:
     # for the project
     che-plugin-registry-directory: che-plugin-registry
     devworkspace-operator-index-disconnected-install: quay.io/devfile/devworkspace-operator-index:release-digest
-    devworkspace-operator-version-patch: "0.21.1"
+    devworkspace-operator-version-patch: "0.25.1"
```

```suggestion
    devworkspace-operator-version-patch: "0.25.0"
```
I do not think there is 0.25.1 version